api/src/main/resources/marquez/db/migration/V43__alter_jobs_add_job_parent_id.sql

```diff
@@ -0,0 +1,44 @@
+ALTER TABLE jobs ADD COLUMN IF NOT EXISTS parent_job_uuid uuid CONSTRAINT jobs_parent_fk_jobs REFERENCES jobs (uuid);
```

Comment:
_Minor: I would also update the migration script to reflect the name of the column added:

```
V43__alter_jobs_add_job_parent_uuid.sql
```